federation/pkg/federation-controller/ingress/ingress_controller.go

```diff
@@ -287,6 +297,9 @@ func (ic *IngressController) reconcileIngress(ingress types.NamespacedName) {
 			// We can't supply server-created fields when creating a new object.
 			desiredIngress.ObjectMeta.ResourceVersion = ""
 			desiredIngress.ObjectMeta.UID = ""
+			ic.eventRecorder.Eventf(baseIngress, api.EventTypeNormal, "CreateInCluster",
+				"Creating ingress in cluster %s", cluster.Name)
```

Comment:
Include the ingress namespace and name?  util.NamespacedName.String() can be used to ensure consistent formatting.